### PR TITLE
Resolves double component name in display name

### PR DESCRIPTION
**Pull request by andrewtpoe**
This PR resolves the [bug](https://github.com/styled-components/babel-plugin-styled-components/issues/73) where the `displayName` and CSS class name include duplicates of the same name when the `componentName` is identical to the `blockName`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.1.7",
```
Comment (philpl) [inline package.json]: Can you remove the version bump please? We'll take care of it when publishing ;)

**Comment by andrewtpoe** [comment]
I've removed the version bump. If it's all the same, I'll leave the squash to you when merging. 😄

File: test/fixtures/05-use-file-name/before.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import styled from "styled-components";
 
 const Test = styled.div`color: red;`;
+const before = styled.div`color: blue;`;
```
Comment (philpl) [inline test/fixtures/05-use-file-name/before.js]: Hm, sorry, one more thing ^^"

Can you add an "inverted" test here? A component that doesn't have the name `before`? This way we'll have both cases covered in this file

Comment (andrewtpoe) [inline test/fixtures/05-use-file-name/before.js]: I'm not sure I follow exactly. Isn't that what `Test` is directly above this component?

Comment (philpl) [inline test/fixtures/05-use-file-name/before.js]: Oh true! Sorry, I must've overlooked it

**Pull request merged**